framework/source/class/qx/ui/form/AbstractSelectBox.js

```diff
@@ -272,11 +282,42 @@ qx.Class.define("qx.ui.form.AbstractSelectBox",
      */
     _onBlur : function(e)
     {
+      // MS Edge only fix for [ISSUE #9174] 
+      if(qx.core.Environment.get("browser.name") == "edge") {
+        if(this.__inhibitBlur === true) {
+          this.__inhibitBlur = false;
+          this.focus();
+          return;
+        }
+      }
+      
       this.close();
     },
 
 
     /**
+     * Handler for the pointerdown event of scrollbar-y child of the list 
+     * MS Edge only fix for [ISSUE #9174] 
+     * 
+     * @param e {qx.event.type.Pointer} The pointerdown event.
+     */
+    __onScrollbarYPointerDown : function(e) {
```

Comment:
Since these two pointer functions are explicitly only for edge, maybe add them in the constructor, in Environment-specific code?

```
if(qx.core.Environment.get("browser.name") == "edge") {
  this.__onScrollbarYPointerDown = function(e) { ... };
  this.__onScrollbarYPointerUp = function(e) { ... };
}
```